src/components/SubTaskList/utils.js

```diff
@@ -1,4 +1,5 @@
-export default (snapValue) => {
+export const getSubtaskListAsArray = (snapValue = {}) => {
+  if (typeof snapValue !== 'object' && snapValue.constructor !== Object) return [];
```

Comment:
if you check if something is not an object, you should use `instanceof`, because `typeof` will give you the result `object` when it is an array and even when it is a function. This is just how JavaScript works.

`!(snapValue instanceof Object)` will give you a definite answer.